htdocs/core/modules/project/doc/doc_generic_project_odt.modules.php

```diff
@@ -482,7 +482,30 @@ function write_file($object,$outputlangs,$srctemplatepath)
 
 				dol_mkdir($conf->projet->dir_temp);
 
-				$socobject=$object->thirdparty;
+				// If PROJECTLEADER contact defined on project, we use it
+				$usecontact=false;
+				$arrayidcontact=$object->getIdContact('external','PROJECTLEADER');
+				if (count($arrayidcontact) > 0)
+				{
+					$usecontact=true;
+					$result=$object->fetch_contact($arrayidcontact[0]);
+				}
+
+				// Recipient name
+				if (! empty($usecontact))
+				{
+					// On peut utiliser le nom de la societe du contact
+					if (! empty($conf->global->MAIN_USE_COMPANY_NAME_OF_CONTACT)) $socobject = $object->contact;
```

Comment:
This test was dedicated to invoice/proposal/order to use contact of thirdparty instead of thirdparty. But here, we want to set project leader whatever is thirdparty (we don't mind the thirdparty here). That's why test on MAIN_USE_COMPANY_NAME_OF_CONTACT is useless. I think you can directly set 
$contactobject = $object->contact;
if (! empty($usecontact)) is true
and always keep the $socobject=$object->thirdparty; oustide this test.